python/mxnet/image/detection.py

```diff
@@ -726,10 +726,23 @@ def _parse_label(self, label):
                 %(str(raw.shape), obj_width)
             raise RuntimeError(msg)
         out = np.reshape(raw[header_width:], (-1, obj_width))
-        # remove bad ground-truths
-        valid = np.where(np.logical_and(out[:, 3] > out[:, 1], out[:, 4] > out[:, 2]))[0]
+
+        # remove invalid ground-truths
+        if ((obj_width - 1) / 4) == 1:
```

Comment:
In fact, it is not safe to check the length of object, since user may append additional info to the objects, not necessarily polygon label.